src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -405,3 +404,56 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
     with self.argument_context('iot central app identity show') as c:
         c.argument('app_name', app_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('iot central app private-endpoint-connection') as c:
+        c.argument('private_endpoint_connection_name', options_list=['--name', '-n'], required=False,
+                   help='The name of the private endpoint connection associated with IoT Central application.'
+                        'Required if --id is not specified')
```

Comment:
The default value of `required` is `False`, so please do not explicitly define `required=False`